test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/transforms/RenameInnerFieldInsideDeferSnapshot.kt

```diff
@@ -0,0 +1,111 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.defer.transforms
```

Comment:
Can we add tests with lists as well? I want to ensure that the path manipulation logic we're introducing works well with list indexes, etc.